pkg/bolt/issueRepo.go

```diff
@@ -59,18 +59,18 @@ func (is *IssueRepo) SaveCommented(id string, commentID string) error {
 	})
 }
 
-func (is *IssueRepo) FindCommentOnIssue(id string, commentID string) (string, error) {
+func (is *IssueRepo) FindCommentOnIssue(id, commentID string) (string, error) {
 	var ret []byte
 	err := is.DB.View(func(tx *bolt.Tx) error {
 		b := tx.Bucket([]byte(commentBucket))
 		if nil == b {
-			return nil
+			return errors.Wrap(err, "bolt issue repo failed to find comment on issue ")
```

Comment:
I think this error should be different. The problem here is the bucket is nil rather than failing to find a comment. So I think we can make the error message clearer and then maybe keep the error below as ```failed to find comment```